src/pineko/theory.py

```diff
@@ -346,6 +347,15 @@ def fk(self, name, grid_path, tcard, pdf):
         do_log = self.activate_logging(
             paths["logs"]["fk"], f"{self.theory_id}-{name}-{pdf}.log"
         )
+        # check if grid contains SV if theory is requesting them
+        xir = tcard["XIR"]
+        xif = tcard["XIF"]
```

Comment:
yes, we have there "activateFact" (or similar name) in order to decide whether to compute the SV grids - however for `pineko` this is not sufficient since we need to collapse on the actual value - or what were you thinking?

Reply:
Sorry, so it is just the comment to be referred to the code below.

Maybe I'd just move it: first `# check if sv are requested` then `# in case of sv, check they are available in the grid`.

Reply:
you mean just the comments right? because the code is already doing that: first it checks if the theory is asking for scale-variations and then it checks whether they are available in the gird.